libsolidity/Types.cpp

```diff
@@ -561,6 +561,21 @@ FunctionType::FunctionType(FunctionDefinition const& _function, bool _isInternal
 	m_location = _isInternal ? Location::INTERNAL : Location::EXTERNAL;
 }
 
+FunctionType::FunctionType(TypePointers const& _parameterTypes, TypePointers const& _returnParameterTypes,
+						   FunctionType::Location _location, bool _gasSet, bool _valueSet):
+	m_parameterTypes(_parameterTypes), m_returnParameterTypes(_returnParameterTypes),
+	m_location(_location), m_gasSet(_gasSet), m_valueSet(_valueSet)
+{
+	if (m_location == Location::EXTERNAL)
```

Comment:
This is weird, seems I stopped in the middle of refactoring. Thanks for spotting!